homeassistant/components/humidifier/strings.json

```diff
@@ -57,10 +57,10 @@
           "state": {
             "normal": "Normal",
             "eco": "Eco",
-            "away": "Away",
+            "away": "[%key:common::state::not_home%]",
```

Comment:
Same comment as above, this state is `away` not `not_home`.

Reply:
Thanks, reverted.